modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormApi.ts

```diff
@@ -8,13 +8,17 @@ import {
 import { InlineContent } from '@ephox/bridge';
 import { Singleton } from '@ephox/katamari';
 
+import * as ContextToolbarFocus from './ContextToolbarFocus';
 import { backSlideEvent } from './ContextUi';
 
 export const getFormApi = <T>(input: AlloyComponent): InlineContent.ContextFormInstanceApi<T> => {
   const valueState = Singleton.value<T>();
 
   return ({
-    setInputEnabled: (state: boolean) => Disabling.set(input, !state),
+    setInputEnabled: (state: boolean) => {
+      ContextToolbarFocus.focusParent(input);
```

Comment:
Do we want to always focus parent or just for disabled state? 🤔

Reply:
Yes technically it would be a noop for setting enabled since it checks if the focus in on the input and if it's disabled it can't have focus. But being explicit of this with an if statement is clearer.